12-offer-encoding.md

```diff
@@ -457,6 +618,9 @@ The writer:
     - otherwise:
       - if it sets `invreq_chain` it MUST set it to bitcoin.
     - MUST set `signature`.`sig` as detailed in [Signature Calculation](#signature-calculation) using the `invreq_payer_id`.
+    - if it sets `invreq_recurrence_cancel`:
+      - SHOULD NOT send any future `invoice_request` for this offer with this `invreq_payer_id` without `invreq_recurrence_cancel` set.
+      - MAY omit `invreq_amount` and `invreq_quantity`.
```

Comment:
Since `invreq_recurrence_cancel` is meant to indicate immediate cancellation (line 751: “MUST NOT send an invoice in reply”), shouldn’t `invreq_amount` and `invreq_quantity` be “MUST NOT present” as they would be meaningless in this context?

Reply:
Okay, that makes sense! Keeping it flexible might reduce the transformation effort when adapting a regular invoice request into a recurrence cancellation.